css/index.css

```diff
@@ -12,23 +12,96 @@
   --btn-color: #396df2;
   --headline-color: #172b4d;
   --content-color: #7a869a;
+  --btn-disabled-color: #5e6c84;
+  --input-border-color: #10f;
+  --textarea-border-color: #0004ff;
   --social-icons-color: #505f79;
   --content-hover-color: #344563;
   --dot-background-color: #c1c7d0;
   --body-background-color: #e5e5e5;
   --vector-background-color: #f7f7f9;
-  --workslink-background-color: #f3f4ff;
   --abt-social-headline-color: #5868f1;
-  --btn-disabled-color: #5e6c84;
+  --btn-hover-background-color: #3346f3;
+  --workslink-background-color: #f3f4ff;
   --btn-active-background-color: #2230d2;
+  --contact-section-background-color: #6070ff;
+  --navbar-menu-li-color: rgba(0 0 0 0.5);
   --btn-hover-box-shadow: rgba(64 83 252 24%);
-  --btn-hover-background-color: #3346f3;
+  --navbar-menu-color: rgba(96 112 255 / 95%);
+  --menu-container-background-color: rgba(0 46 252 0.5);
 }
 
 body {
   background-color: var(--body-background-color);
 }
 
+button:hover {
+  cursor: pointer;
+}
+
+.navbar-menu {
+  opacity: 0;
+  width: 100%;
+  height: 100vh;
+  background: var(--navbar-menu-color);
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 100%;
+  padding: 8rem 2rem;
+}
+
+.navbar-menu-ul {
+  flex-direction: column;
+}
+
+.navbar-background {
+  width: 100%;
+  height: calc(100vh - 75vh);
+}
+
+.navbar-menu-li {
+  margin-bottom: 40px;
+}
+
+.navbar-menu-li a {
+  color: var(--primary-color);
+  display: block;
+  font-size: 30px;
+  font-weight: 600;
+}
+
+.navbar-menu-li a:hover {
+  cursor: pointer;
+  color: rgb(128 255 0);
+  background-color: #757980;
+}
+
+.navbar-menu-li a:active {
+  cursor: pointer;
+  color: var(--navbar-menu-li-color);
+}
```

Comment:
Let's remove all form of compound selector, using simple selectors make it easier to overwrite styles.